tests/test_contrib/test_scatter.py

```diff
@@ -152,12 +160,21 @@ def test_alpha_param(self):
         Test that the user can supply an alpha param on instantiation
         """
         # Instantiate a scatter plot and provide a custom alpha
-        visualizer = ScatterVisualizer(
-            x="temperature", y="light", alpha=0.7
-        )
+        visualizer = ScatterVisualizer(alpha=0.7, features=["a", "b"])
 
+        # Test param gets set correctly
         assert visualizer.alpha == 0.7
 
+        # Mock ax and fit the visualizer
+        visualizer.ax = mock.MagicMock(autospec=True)
+        visualizer.fit(self.X[:, :2], self.y)
+
+        # Test that alpha was passed to the scatter plot
+        _, scatter_kwargs = visualizer.ax.scatter.call_args
+        assert "alpha" in scatter_kwargs
+        assert scatter_kwargs["alpha"] == 0.7
```

Comment:
Generally speaking, you can use the `assert_called_with` and `assert_called_once_with` functions to ensure that `ax.scatter` was called correctly. However, in this case you'd have to know all the arguments that were passed to the function. You could use `mock.ANY` so you don't necessarily have to know what value was passed, but you'd still have to know the arguments list. This method specifically checks for `alpha`, which is what we're testing in this function.

Reply:
Nice - this is also something we should earmark for #558